src/clr-angular/popover/tooltip/_tooltips.clarity.scss

```diff
@@ -86,6 +86,7 @@
             opacity: 0;
             transition: opacity 0.3s linear;
             white-space: normal;
+            word-wrap: break-word;
```

Comment:
Is this something we want on by default? Or would it make more sense to offer a utility class that applies it to the tooltip?

Reply:
I agree, best UX is to avoid putting more content and making this the default subverts that slightly. I would make an optional class to wrap, or just put some documentation for people to know how to override in the future. Not sure we really should proactively encourage this?

Reply:
@mathisscott @gnomeontherun: 
Just read the description here: https://github.com/vmware/clarity/issues/1146
Will close this PR. It seems like we decided not to add `word-break` in our CSS at all and let the user add it at their end if they are facing such issues because it breaks the word.

I do feel that this style can be included because this won't affect how the tooltip content is rendered in general. If there are multiple words in one line which exceed the width of the content, then the entire last word will be wrapped. It won't be broken into two parts.

Eg: 
![image](https://user-images.githubusercontent.com/1426805/36800243-0ca68d8c-1c7d-11e8-9991-b4a31dcff047.png)

`word-break` just makes the styling more fool proof and avoids errors like the ones mentioned in #2020 

For now I will close this PR but may be we can talk about this later on.

Reply:
I'm fine including it. I just wondered if it needed to be the default...

Reply:
I am going to route this back to UX and see what they say about this. Will reopen this PR if they agree too. Thanks!